week_1/project/week_1.py

```diff
@@ -50,16 +51,25 @@ def get_s3_data(context):
     return output
 
 
-@op
-def process_data():
-    pass
+@op(
+    ins={"stock_list": In(dagster_type=List[Stock])},
+    out={"agg": Out(dagster_type=Aggregation)},
+    description="Find the highest high value from a list of stocks"
+    )
+def process_data(stock_list):
+    high_stock = max(stock_list, key=attrgetter('high'))
```

Comment:
Nice max/key pythonic line. I used a lambda, but the attrgetter is even quicker. I wonder with a package like pydantic if there may be an even simpler `key='high'` method, but this is the first time I've used it.

Reply:
Thanks, Chris! It had been awhile since I wrote python, so a quick google of _"get max value of attribute python"_ helped with that one. Lambdas work great too.